descriptors/ModuleDescriptor-template.json

```diff
@@ -4,7 +4,7 @@
   "provides": [
     {
       "id": "circulation",
-      "version": "7.4",
+      "version": "7.5",
```

Comment:
Why has the circulation interface version changed?

There does not appear to be any changes to the interface definition?

Reply:
There are no changes in the interface definition, but additional calls (mod-config) have been added to API for `check-out` and `renewal`

Reply:
The additional calls are IMO implementation aspects of this change, rather than interface. 

I’m aware that the module descriptor is a grey area, as it binds an interface to an implementation and describes endpoint permissions.

To me, the permissions granted are part of the binding and not the interface.

Does that make sense?

If there are no interface changes, please revert this incrementing of the interface version here and in the RAML.

Reply:
ok

Reply:
done